rpc/server.go

```diff
@@ -176,6 +182,7 @@ func (conn *Conn) Start() {
 	conn.mutex.Lock()
```

Comment:
Wouldn't it be more appropriate for this method to take a Context as an argument rather than creating one from scratch? (or perhaps pass the context into NewConn)

Reply:
Yes, I'll update to pass it into Start.